ci-operator/step-registry/aws/provision/vpc/shared/aws-provision-vpc-shared-commands.sh

```diff
@@ -757,6 +757,14 @@ if [[ "${ADDITIONAL_SUBNETS_COUNT}" -gt 0 ]]; then
   aws_add_param_to_json "AdditionalSubnetsCount" ${ADDITIONAL_SUBNETS_COUNT} "$vpc_params"
 fi
 
+# aws efs single zone cross account the shared vpc subnet zone should be consistent with the cluster
+if [[ ${ENABLE_SINGLE_ZONE} == "true" ]] && [[ "${ZONES_COUNT}" -eq 1 ]]; then
```

Comment:
Usually, `aws-provision-vpc-shared` is invoked in pre-install phase, `ENABLE_SINGLE_ZONE` may cause others confusion. As this is a particular case for EFS, can we add a `EFS` keyword in the name? like `EFS_ENABLE_SINGLE_ZONE`, and add a clear comment that this flag is used by `storage-conf-csi-optional-aws-efs-cross-account`